wiki/Glossary/en.md

```diff
@@ -34,21 +34,23 @@ Authentic denotes a beatmap that emulates in every aspect of a specific stage fr
 _Related: [BanchoBot](/wiki/BanchoBot)_
 
 Bancho is the easily expandable server coded by peppy.
-Bancho replaces the need for a third-party IRC server in the management of all online content in osu!.
+Bancho replaces the need for a third-party [IRC](/wiki/IRC) server in the management of all online content in osu!
 Logging with an osu!account ingame will automatically attempt to connect to Bancho until done successfully.
 
 ### Beat
 
 Beat is a term that applies to the timeline in editor mode.
 It is a period of time that songs can be divided into based off of their BPM.
-A correctly timed beatmap will have each beat correspond to the song's rhythm and a whole beat should fall between two white lines on the timeline.
+A correctly timed beatmap will have each beat correspond to the song's rhythm and a whole beat should fall between two white lines on the [timeline](/wiki/Timeline).
```

Comment:
the singular is a redirect... (this is talking about the timing mode's timeline I think